cockpit/leapp.html

```diff
@@ -141,10 +141,10 @@
 
         function call_leapp(cmd_args) {
             return cockpit.spawn(
-                ["/opt/leapp/bin/leapp-tool"].concat(cmd_args),
+                ["/usr/bin/leapp-tool"].concat(cmd_args),
```

Comment:
The symlink creation instructions in the README (that cover running the demo from a checkout rather than an installed package) will need to be updated for this change, as will the integration tests (the plugin integration test isn't run by default, which is why the CI isn't complaining about it, but `behave --wip` will run it locally).